docs/docs/welcome.mdx

```diff
@@ -0,0 +1,90 @@
+---
+slug: /
+title: Welcome to Nebari's documentation
+---
+
+export const invisible_table = {
+   color: 'blue',
+   lineHeight: 10,
+   padding: '1.5em',
+   background: 'white',
+   backgroundColor: 'white',
+}
+
+:::warning We are currently undergoing a rename from [QHub](https://docs.qhub.dev/) to Nebari.   
+:construction: Please excuse our dust as we build out our new docs :) :construction:
+:::
+
+[INSERT NEBARI LOGO HERE]
+
+Nebari: The Roots of a Bonsai Tree
+
+## Nebari Vision
+
+An opinionated open source [JupyterHub][jupyterhub] distribution that provides 
+an all-in-one development and experimentation platform for teams to work 
+efficiently and collaboratively.
+
+Deployment, integration and maintenance is minimized by using the 
+infrastructure as code (IaC) management approach.
+
+## Nebari has a *DevOps for non-DevOps people* approach.
+
+![Nebari overview sequence](/img/nebari_overview_sequence.png)
+
+### Components
+
+The technology stack is an integration of the following existing open source libraries:
+
+- [**Terraform**](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.
+- [**Kubernetes**](https://kubernetes.io/docs/home/): a cloud-agnostic orchestration system
+- [**Helm**](https://helm.sh/): a package manager for Kubernetes
+- [**JupyterHub**](https://jupyter.org/hub): a shareable compute platform for data science
+- [**JupyterLab**](https://jupyterlab.readthedocs.io/en/stable/): a web-based interactive development environment for Jupyter Notebooks
+- [**Dask**](https://docs.dask.org/en/latest/): a scalable and flexible library for parallel computing in Python
+  - [**Dask-Gateway**](https://gateway.dask.org/): a secure, multi-tenant server for managing Dask clusters
+- [**Keycloak**](https://www.keycloak.org/): Open Source Identity and Access Management
+- [**GitHub Actions**](https://docs.github.com/en/actions): a tool to automate, customize, and execute software development workflows in a GitHub repository.
+- [**traefik**](https://traefik.io/): for routing web/tcp traffic inside cluster
+  - [**traefik-forward-auth**](https://github.com/thomseddon/traefik-forward-auth): for single sign on and easy securing of web applications
+
+Amongst the newly created open source libraries on the tech stack are:
+
+- [**Jupyterhub SSH**](https://github.com/yuvipanda/jupyterhub-ssh): brings the SSH experience to a modern cluster manager.
+- [**Jupyter Videochat**](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.
```

Comment:
@dharhas - we discussed getting this out of nebari *soon* shall we remove the mention?